src/bptl/work_units/zgw/tasks/zaak_relations.py

```diff
@@ -310,6 +311,44 @@ def perform(self) -> Optional[dict]:
             request_headers=headers,
         )
 
+        try:
+            bijdrage_aard_omgekeerde_richting = check_variable(
+                variables, "bijdrageAardOmgekeerdeRichting", empty_allowed=True
+            )
+            if (
+                bijdrage_aard_omgekeerde_richting
+                and bijdrage_aard_omgekeerde_richting not in AardRelatieChoices.values
+            ):
+                raise ValueError(
+                    f"Unknown 'bijdrageAardOmgekeerdeRichting': '{bijdrage_aard_omgekeerde_richting}'"
+                )
+        except MissingVariable:
+            # To avoid having to edit BPMN models - default of bijdrage_aard_omgekeerde_richting
+            # is "onderwerp" if it isn't explicitly given in the variables.
+            bijdrage_aard_omgekeerde_richting = AardRelatieChoices.onderwerp
+
+        # Relating of secondary zaken to their main zaak.
+        if (
+            bijdrage_aard != AardRelatieChoices.onderwerp
+            and bijdrage_aard_omgekeerde_richting
```

Comment:
```suggestion
            bijdrage_aard != AardRelatieChoices.onderwerp
            
```
`bijdrage_aard_omgekeerde_richting` always has a value (from task variables or the default one)

Reply:
Well, if it is explicitly empty (""), then we don't want to add the reverse relation.

Reply:
oh right!